cmd/restic/cmd_restore.go

```diff
@@ -1,12 +1,13 @@
 package main
 
 import (
+	"strings"
+
 	"github.com/restic/restic/internal/debug"
 	"github.com/restic/restic/internal/errors"
 	"github.com/restic/restic/internal/filter"
 	"github.com/restic/restic/internal/restic"
 	"github.com/restic/restic/internal/restorer"
-	"strings"
```

Comment:
This change is unrelated and should be removed. I can try to do that when I have some time if you are fed up with making corrections to this PR by now :)

Reply:
Fixed it, I think it was go fmt or something that changed it.